### ISSUE TITLE
Documentation not clear on how to add accessibility options to snackbar

### ISSUE THREAD

**Issue by omaracrystal** [material-components/material-components#60]
Documentation not clear on how to add accessibility options to snackbar.

I am using the Material Design snackbar notification and have the duration set for 99999 for now (would prefer an infinite option if there is one). On the snackbar we have two actions "close" and a link to another page. I have tabindex=0 on this component template that's within the snackbar... however the keyboard users cannot access anything within the snackbar. 

On your site: https://material.io/develop/web/components/snackbars/#screen-readers
I read this:

```
Dismiss Icon
Snackbars are intended to dismiss on their own after a few seconds, but a dedicated dismiss icon may be optionally included as well for accessibility purposes.
```

How do you set the dedicated dismiss icon? What does this mean?

**Comment by ddemaree** [comment]
Thanks for opening this issue; sorry for the delay responding.

Per the docs for `mdc-snackbar`, you can add the explicit dismiss icon by including a `mdc-snackbar__dismiss` sub-element inside the snackbar markup. There are [some examples in our demo catalog](https://material-components.github.io/material-components-web-catalog/#/component/snackbar). 

@abhiomkar Can you answer the keyboard accessibility question?

**Comment by abhiomkar** [comment]
Thanks for the question! Somehow this post missed our radar.

We've a tracking bug (Issue #153) to add infinite duration to snackbar.

To answer your question related to dismiss button, as David mentioned you would need to use `mdc-snackbar__dismiss` element inside snackbar template to render dismiss icon. You wouldn't need tabindex on root element of snackbar.

```html
<button class="mdc-icon-button mdc-snackbar__dismiss material-icons" title="Dismiss">close</button>
```

Snackbar doesn't grab focus on open, user would've to manually TAB through the snackbar action buttons to reach focus.

It would be easier for us to investigate if you can share reduced example in codepen to demonstrate keyboard issue.

**Comment by omaracrystal** [comment]
> Thanks for the question! Somehow this post missed our radar.
> 
> We've a tracking bug (Issue #153) to add infinite duration to snackbar.
> 
> To answer your question related to dismiss button, as David mentioned you would need to use `mdc-snackbar__dismiss` element inside snackbar template to render dismiss icon. You wouldn't need tabindex on root element of snackbar.
> 
> ```
> <button class="mdc-icon-button mdc-snackbar__dismiss material-icons" title="Dismiss">close</button>
> ```
> 
> Snackbar doesn't grab focus on open, user would've to manually TAB through the snackbar action buttons to reach focus.
> 
> It would be easier for us to investigate if you can share reduced example in codepen to demonstrate keyboard issue.

Awesome! Thanks!

**Comment by cailinw** [comment]
Are there any updates on adding support for infinite duration of snackbar? I think this would be a great feature.

**Comment by leticiarossi** [comment]
Please see https://github.com/material-components/material-components-web/issues/153
